src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,86 @@
+package duke;
+
+import duke.Deadline;
+import duke.Event;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    Path currentRelativePath = Paths.get("");
+    String s = currentRelativePath.toAbsolutePath().toString();
```

Comment:
Is there any reason these are non-private? Does `Storage` have subclasses that require access to these? On the other hand, even if there are, perhaps you could consider using getter methods even for only subclasses to use.

Reply:
Since `s` is a variable with a large scope (a class instance variable), maybe consider using a longer, more descriptive name for it, or perhaps consider moving it to a smaller scope.

Reply:
Yes, I forgot to put the access modifier `private`. And you suggestion for the naming is correct. However, these fields are not really used anywhere, so I probably should delete them. Thank you for the suggestions.